src/features/student/course/Exam.module.scss

```diff
@@ -0,0 +1,26 @@
+@import '../../../colors.module.scss';
+
+.examContainer {
+    display: flex;
+    justify-content: space-between;
+    background-color: $colorSecondary;
+    padding: 10px 15px;
+    margin: 5px 0;
+    border-radius: 12px;
+}
+
+.examInputContainer {
+    display: flex;
+    padding: 5px 0;
+    width: 50px;
+}
+
+.examLabelContainer {
+    display: flex;
+    align-items: center;
+    flex-grow: 1;
+}
+
+.examInput {
+    text-align: center !important;
```

Comment:
Try to avoid !important. Normally when your style does not work, it is because it has to low specificity. A good solution is to make your class more specific, for example by adding more specific classes like the antd classes.

But I removed the !important and did not see any difference, are you sure the text-aling center is needed?